src/components/BulkEnrollmentPage/BulkEnrollment.scss

```diff
@@ -52,3 +52,13 @@
     overflow: scroll;
   }
 }
+
+.data-table-selector-column-wrapper {
```

Comment:
did you check this looks great under different browser widths? since fixed width I figure we are okay but doesn't hurt to check.

Reply:
good q! I was curious myself-

**Before:**
**mobile**
![](https://files.slack.com/files-pri/TB6DAA5V5-F027M4LBL8P/image.png)

**small desktop**
![](https://files.slack.com/files-tmb/TB6DAA5V5-F02882YLUMS-6c94aba1d4/image_720.png)


**After**
**mobile**
![](https://files.slack.com/files-pri/TB6DAA5V5-F027V27Q1JS/image.png)

**small desktop**
![](https://files.slack.com/files-pri/TB6DAA5V5-F027M254W5D/image.png)